docs/biosample.rst

```diff
@@ -30,24 +31,36 @@ genomic array as well as RNA-seq experiments) may refer to the same Biosample.
      - string
      - recommended
      - arbitrary identifier
+   * - derived_from_id
+     - string
+     - optional
+     - id of the biosample from which the current biosample was derived (if applicable)
    * - description
      - string
      - optional
      - arbitrary text
    * - sampled_tissue
      - :ref:`rstontologyclass`
-     - required
+     - optional
      - Tissue from which the sample was taken
+   * - sample_type
+     - :ref:`rstontologyclass`
+     - optional
+     - type of material, e.g., RNA, DNA, Cultured cells
    * - phenotypic_features
-     - :ref:`rstphenotypicfeature`
+     - :ref:`rstphenotypicfeature` (List)
      - recommended
      - List of phenotypic abnormalities of the sample
+   * - measurements
+     - :ref:`rstmeasurement` (List)
+     - optional
+     - List of measurements of the sample
    * - taxonomy
      - :ref:`rstontologyclass`
      - optional
      - Species of the sampled individual
    * - individual_age_at_collection
```

Comment:
Think this would be better being `time_of_collection`? This is a TimeElement so could be an age, timestamp or time interval.

Reply:
I agree time of collection is better. I will change this in the documentation!